.github/workflows/push-deploy.yml

```diff
@@ -21,11 +21,8 @@ jobs:
         SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
         SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
     - name: Upload release asset
-      uses: actions/upload-release-asset@v1
+      uses: softprops/action-gh-release@v2
```

Comment:
Realizing I just approved/merged in some other repos, but curious if we need to similarly update other actions?
- https://github.com/10up/insert-special-characters/blob/develop/.github/workflows/build-release-zip.yml
- https://github.com/10up/insert-special-characters/blob/develop/.github/workflows/push-asset-readme-update.yml

Reply:
@Sidsector9 mind pushing a fix for the action examples then?

Reply:
Opened a PR here - https://github.com/10up/action-wordpress-plugin-deploy/pull/150